src/backend/joanie/tests/payment/base_payment.py

```diff
@@ -0,0 +1,33 @@
+"""Common Base Payment backend test"""
+from django.core import mail
+from django.test import TestCase
+
+
+class BasePaymentTestCase(TestCase):
+    """Common method to test the Payment Backend"""
+
+    def _check_purchase_order_email_sent(self, email, username, order):
+        """Shortcut to check purchase order email has been sent"""
+        # check email has been sent
+        self.assertEqual(len(mail.outbox), 1)
+
+        # check we send it to the the right email
+        self.assertEqual(mail.outbox[0].to[0], email)
```

Comment:
What do you think to put this code outside then pass the mail to check in params ?

```suggestion
        # check we send it to the the right email
        self.assertEqual(mail.to[0], email)
```

Reply:
I mean yes we could, but I think by passing the detailed parameters from the test and leaving this inside is ok. What bothers you with this way of doing it?

Reply:
got it! don't think testing the payment mail, this point could be a concern though